docs/terraformoptions.adoc

```diff
@@ -223,7 +223,7 @@ e.g.
 |"vcnmodule"
```

Comment:
Please also change the default name to "vcn"

Reply:
Added the missing default for `vcn_name` below this - these line up with the variables now; let me know if we're wanting to change further in this PR.